config/src/config/network_config.rs

```diff
@@ -200,6 +201,23 @@ pub struct SeedPeersConfig {
     pub seed_peers: HashMap<PeerId, Vec<NetworkAddress>>,
```

Comment:
why not a tuple struct?

Reply:
i think it's so the actual seed_peers.toml looks nicer

```toml
# seed_peers.toml

[seed_peers]
# each seed peer info
```